docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -33,7 +33,7 @@ notebooks:
     publish: <Boolean>
 
     # An optional array of absolute paths to other files in the Git repository that should be read and included with the notebook.
-    otherFiles: <String Array>
+    otherFiles: <String Array>  # Must be a YAML array of strings
```

Comment:
Honestly, I'd often have to look up array syntax when switching languages. 
I think a template example is more intuitive since a user can copy-paste and replace
i.e.
```
otherFiles:
  - /exampleFile1
  - /exampleFile2
```

Reply:
We can do something like that, but these `<String Arrays>` appear in the "generic" template, we have "filled out" examples below on the same page.  It wouldn't make sense to have actual paths in the generic example.  So, maybe something like:
```
otherFiles:
   - <String>
   - <String>
```
And if it's as above, we might be able to delete the advice about array formats that I added, because we're demonstrating it above.

Reply:
```
otherFiles:
   - <String>
   - <String>
```

This is also how we do it with authors (see my other comment). I like this proposal.